iot/api-client/generate_keys.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
For portability (windows) I'm wondering if it's easy to do this in Windows? No action needed, but just curious.